plenum/config.py

```diff
@@ -406,5 +406,5 @@
 TXN_AUTHOR_AGREEMENT_AML_CONTEXT_LIMIT = MSG_LEN_LIMIT - 2048
 
 # TAA acceptance time valid deviations (secs)
-TXN_AUTHOR_AGREEMENT_ACCEPANCE_TIME_BEFORE_TAA_TIME = 120
+TXN_AUTHOR_AGREEMENT_ACCEPTANCE_TIME_BEFORE_TAA_TIME = 120
 TXN_AUTHOR_AGREEMENT_ACCEPANCE_TIME_AFTER_PP_TIME = 120
```

Comment:
The same typo is in `TXN_AUTHOR_AGREEMENT_ACCEPANCE_TIME_AFTER_PP_TIME ` as well

Reply:
thanks, fixed